### PR TITLE
Button component added

### PR DESCRIPTION
**Pull request by MihirNS**
Ripple effect is not working for both components right now.

### CODE REVIEW

File: src/Checkbox.jsx [hunk 1]
```diff
@@ -3,17 +3,11 @@ import {Set as ImmutableSet, Map as ImmutableMap} from 'immutable'
 import {getCorrectEventName} from '@material/animation/dist/mdc.animation'
 import {MDCCheckboxFoundation} from '@material/checkbox/dist/mdc.checkbox'
 import RippleFoundation from './RippleFoundation.jsx'
-import PropTypes from 'prop-types';
-import '@material/checkbox/dist/mdc.checkbox.css';
-
-function getMatchesProperty(HTMLElementPrototype) {
-  return ['webkitMatchesSelector', 'msMatchesSelector', 'matches'].filter((p) => p in HTMLElementPrototype).pop();
-}
+import PropTypes from 'prop-types'
```
Comment (pratikgaloria) [inline src/Checkbox.jsx]: Why we are not using PropTypes from React itself?

Comment (MihirNS) [inline src/Checkbox.jsx]: because it has been deprecated 

https://facebook.github.io/react/docs/typechecking-with-proptypes.html

**Comment by pratikgaloria** [comment]
We should add eslint for linting the files, this would keep our code clean and standard. We can follow Airbnb standards. If you are fine with that, I will configure it.

**Comment by MihirNS** [comment]
Yep. I am fine with it.

File: src/Checkbox.jsx (hunk 1 above)
Comment (pratikgaloria) [inline src/Checkbox.jsx]: Oh ok.

**Pull request merged**